modules/038-registry/hooks/checker/hook_loop.go

```diff
@@ -101,19 +101,17 @@ var _ = sdk.RegisterFunc(
 					containers := deployment.Spec.Template.Spec.Containers
 					initContainers := deployment.Spec.Template.Spec.InitContainers
 
-					ret := deckhouseImagesModel{
-						InitContainers: make(map[string]string),
-						Containers:     make(map[string]string),
-					}
+					ret := make(map[string]string, len(containers)+len(initContainers))
 
 					for _, c := range initContainers {
-						ret.InitContainers[c.Name] = c.Image
+						info := fmt.Sprintf("deckhouse/init-containers/%v", c.Name)
```

Comment:
It looks like we're don't want to split info generation logic across multiple files, please return to previous one

Reply:
for `inputs.ImagesInfo.DeckhouseContainerImageRef` you may just handle `deckhouse` container in `if` block in loop